packages/node/src/schematics/application/application.ts

```diff
@@ -144,19 +144,35 @@ function addProxy(options: NormalizedSchema): Rule {
     const projectConfig = getProjectConfig(host, options.frontendProject);
     if (projectConfig.architect && projectConfig.architect.serve) {
       const pathToProxyFile = `${projectConfig.root}/proxy.conf.json`;
-      host.create(
-        pathToProxyFile,
-        JSON.stringify(
-          {
-            '/api': {
-              target: 'http://localhost:3333',
-              secure: false,
+
+      if (!host.exists(pathToProxyFile)) {
+        host.create(
+          pathToProxyFile,
+          JSON.stringify(
+            {
+              [`${options.name}`]: {
```

Comment:
I believe that for the first application we should keep the existing `/api` as key.

Reply:
Done